recipe/meta.yaml

```
@@ -187,9 +187,9 @@ outputs:
 {% endif %}
     test:
       downstreams:
-        # Temporarily disable dowmstream tests on aarch64 as the solver causes timeouts on drone
-        - cython  # [not aarch64]
-        - setuptools  # [not aarch64]
+        # Temporarily disable downstream test for setuptools due to breaking changes
+        - cython
+        # - setuptools
```

Would keep the selector and just comment this line

```suggestion
        # - setuptools  # [not aarch64]
```

---

I commented out the whole line because AFAICT, the setuptools-breakage is not confined to aarch/ppc

---

Currently yes. Though if in the future the `setuptools` issue is fixed, but the `aarch64` issue persists, it may be helpful for the next person who comes here to back things out to the last known working state

---

I don't expect the drone timeouts to be relevant anymore, so resolving this.